TezosKit/Conseil/GetSentTransactionsRPC.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright Keefer Taylor, 2019.
+
+// swiftlint:disable line_length
+
+/// An RPC which fetches sent transactions from an account.
+public class GetSentTransactionsRPC: RPC<[[String: Any]]> {
+  /// - Parameters:
+  ///   - account: The account to query.
+  ///   - limit: The number of items to return.
+  ///   - apiKey: The API key to send in the request headers.
+  ///   - platform: The platform to query.
+  ///   - network: The network to query.
+  public init?(account: String, limit: Int, apiKey: String, platform: ConseilPlatform, network: ConseilNetwork) {
+    guard let escapedPlatform = platform.rawValue.addingPercentEncoding(
+                withAllowedCharacters: CharacterSet.urlQueryAllowed
+          ),
+          let escapedNetwork = network.rawValue.addingPercentEncoding(
+                withAllowedCharacters: CharacterSet.urlQueryAllowed
+      ) else {
+        return nil
+    }
+    let endpoint = "/v2/data/\(escapedPlatform)/\(escapedNetwork)/operations"
+    let headers = [
+      Header.contentTypeApplicationJSON,
+      Header(field: "apiKey", value: apiKey)
+    ]
+    let payload = """
+      {"fields": [],"predicates": [{"field": "kind","set": ["transaction"],"operation": "eq","inverse": false}, {"field": "source","set": ["\(account)"],"operation": "eq","inverse": false}],"orderBy": [{"field": "timestamp","direction": "desc"}],"limit": \(limit)}
```

Comment:
Is this suppose to be valid JSON?  If so why not use Swift's JSON API or SwiftyJSON to build it in a more typesafe manner?

Reply:
Yeah, it is valid JSON.

I think I need a better JSON solution longer term. A follow up PR here will refactor these keys into a logical constant hierarchy. 

I didn't know about SwiftyJSON. I'll try to integrate that wholesale in a coming PR.